common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -18,6 +18,14 @@ import NextLink from 'next/link';
 import { removeEmptyProps } from '../../../utils/json';
 import { useRouter } from 'next/router';
 import ConditionalWrapper from '../ConditionalWrapper/ConditionalWrapper';
+import BetaBar from '@weco/common/views/components/BetaBar/BetaBar';
+
+const BetaBarContainer = styled.div`
```

Comment:
Shouldn't this live with the `BetaBar`?

Reply:
No, this is about where it's positioned in relation to the tabs, it wouldn't apply universally

Reply:
Can't say I 100% understand what the styling is doing. Any chance of leaving a note¿